app/src/main/java/fr/free/nrw/commons/nearby/PlaceRenderer.java

```diff
@@ -117,7 +120,7 @@ protected void hookListeners(View view) {
                 }
             }
             if (onBookmarkClick == null) {
-                ((NearbyFragment) fragment.getParentFragment()).centerMapToPlace(place);
+                ((NearbyParentFragment) fragment.getParentFragment()).centerMapToPlace(place);
```

Comment:
Again, the fragment should not be called from another class to access `centerMapToPlace`. Ideally there could be a interface or event publishing.